### PR TITLE
Adding warning to nu_plugin_formats commands

### LINKED ISSUE #1091: `from ini` doesn't work as documentation says
### Describe the bug

The [documentation ](https://www.nushell.sh/commands/docs/from_ini.html) mentions a command called `from ini`, but it doesn't work unless you install a custom plugin, which is nowhere to be found except in a hidden away [release note](https://www.nushell.sh/blog/2023-02-21-nushell_0_76.html#breaking-changes). 

This was done in https://github.com/nushell/nushell/pull/7942 but it seems the documentation has not been updated.

### How to reproduce

1. Install nushell
2. Use the `from ini` command
3. Get error mentioning `extra positional argument`

### Expected behavior

Expected `from ini` to work as documented

### Screenshots

_No response_

### Configuration

| key                | value                                          |
| ------------------ | ---------------------------------------------- |
| version            | 0.85.0                                         |
| branch             |                                                |
| commit_hash        |                                                |
| build_os           | linux-x86_64                                   |
| build_target       | x86_64-unknown-linux-gnu                       |
| rust_version       | rustc 1.72.0 (5680fa18f 2023-08-23) (Homebrew) |
| cargo_version      | cargo 1.72.0 (26bba4830 2023-08-26)            |
| build_time         | 2023-09-19 18:42:47 +00:00                     |
| build_rust_channel | release                                        |
| allocator          | mimalloc                                       |
| features           | dataframe, default, sqlite, trash, which, zip  |
| installed_plugins  |                                                |

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by TWSiO**
Resolves #1091

In `make_docs.nu` command page generation section, added a check for the commands from the plugin `nu_plugin_formats` (`from ini` and others) to add a warning box to those pages informing the users they have to install and register the plugin to use those commands. Also added the changes to the generated docs as well.

nushell-from-ini-doc

There might be a small merge conflict with #1183, but it should be easy to resolve.

---

I tried to figure out a way to not hardcode the command names in `make_docs.nu` and I figured out how the `make_docs.nu` could determine if a command was from a plugin, however I couldn't figure out how to determine which plugins the respective commands came from which I figured was important to add. Maybe it could parse the nu script at `$nu.plugin-path`, but that seemed kind of janky. If anyone knows how to determine them in that script or has any other ideas, just let me know.

### CODE REVIEW

**Comment by fdncred** [comment]
How about rewording a tiny bit. `Command <command> resides in nu_plugin_formats. In order to use this command, you must install/compile and register nu_plugin_formats.` Something along those lines.

**Comment by TWSiO** [comment]
Reworded

**Comment by fdncred** [comment]
Thanks!
